source/javascripts/services/cookie.mjs

```diff
@@ -0,0 +1,29 @@
+export const COOKIE_NAME = 'analytics_consent'
+
+export const CONSENT_STATUS = {
+  GRANTED: 'CONSENT_GRANTED',
+  DENIED: 'CONSENT_DENIED',
+  UNKNOWN: 'CONSENT_UNKNOWN'
+}
+
+export function loadConsentStatus () {
+  const cookies = document.cookie ? document.cookie.split('; ') : []
+  cookies.forEach(function (rawCookie) {
+    cookie = cookies[i].split('=')
```

Comment:
This'll break because you don't have access to `i` in this forEach - think it should be:
```suggestion
    const cookie = rawCookie.split('=')
```
(sorry, this one was my fault)

Reply:
Thanks for all your comments on this PR - they were really useful and things I didn't see 🙏